widget/settings/pages/config/notifications/index.ts

```diff
@@ -45,6 +45,12 @@ export const NotificationSettings = (): Scrollable<Child, Attribute> => {
                     subtitle: 'The notification will follow the monitor of your cursor',
                     type: 'boolean',
                 }),
+                Option({
+                    opt: options.notifications.clearDelay,
+                    title: 'Clear Delay',
+                    subtitle: 'The delay in milliseconds before a notification is cleared',
+                    type: 'number',
```

Comment:
```suggestion
                    type: 'number',
                    increment: 20,

```
We can add a default increment size of 20 so the button is easier to configure (20ms per click).